src/public/map.js

```diff
@@ -356,21 +386,21 @@ function formatPopup(properties) {
     }
     str += "</UL>";
     str += "<br>";
-  } else if (properties.features && properties.features.length === 0) {
+    }
+   else if (properties.features && properties.features.length === 0) {
```

Comment:
Feel free to ignore me - this probably makes very little difference, but line 390 is off by one space.... 😇